src/modules/components/proposal/proposalVoting/proposalVotingStageStatus/proposalVotingStageStatus.tsx

```diff
@@ -49,6 +49,33 @@ const statusToSecondaryText = (copy: ModulesCopy): Partial<Record<ProposalStatus
     [ProposalStatus.EXPIRED]: copy.proposalVotingStageStatus.secondary.expired,
     [ProposalStatus.UNREACHED]: copy.proposalVotingStageStatus.secondary.unreached,
     [ProposalStatus.VETOED]: copy.proposalVotingStageStatus.secondary.vetoed,
+    [ProposalStatus.EXECUTABLE]: copy.proposalVotingStageStatus.secondary.executable,
+    [ProposalStatus.EXECUTED]: copy.proposalVotingStageStatus.secondary.executed,
+});
+
+const statusToStatusText = (
```

Comment:
Minor naming suggestion
```suggestion
const statusToText = (
```